src/Missings.jl

```diff
@@ -1,7 +1,8 @@
 module Missings
 
 export allowmissing, disallowmissing, ismissing, missing, missings,
-       Missing, MissingException, levels, coalesce, passmissing, nonmissingtype
+       Missing, MissingException, levels, coalesce, passmissing, nonmissingtype,
+       skipmissings, SkipMissings
```

Comment:
Base doesn't export `SkipMissing`, so let's do the same here (this will affect the result of `show`, so be careful to update it).
```suggestion
       skipmissings
```